sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.aest.reissueMSFAA.e2e-spec.ts

```diff
@@ -349,6 +349,40 @@ describe("ApplicationAESTController(e2e)-reissueMSFAA", () => {
     }
   }
 
+  it("Should not reissue an MSFAA when user is not a business administrator.", async () => {
+    // Arrange
+    const student = await saveFakeStudent(db.dataSource);
```

Comment:
I believe that the data setup is not needed for this case. The decorator should stop the access before even reaching the DB.

Reply:
ok. removed the setup.